.github/workflows/bump-jslib.yml

```
@@ -0,0 +1,74 @@
+name: Bump jslib
+
+on:
+  workflow_dispatch:
+
+jobs:
+  bump-jslib:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
```

```suggestion
        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
```

To mitigate some risks of some supply chain attacks, we should use the commit hash of the Action instead of the version